### PR TITLE
[TFAC-1023]: Test out a stronger "supporting" statement on the new tab page

### PR DESCRIPTION
**Pull request by spicermatthews**
We want to test a more direct statement on the tab v4 page “chip”:

Added the new field to cause, built out new experiment.

### CODE REVIEW

File: graphql/database/experiments/features.js [hunk 1]
```diff
@@ -427,6 +428,43 @@ const features = {
...
+            $eq: 'Experiment',
+            $exists: true,
+          },
+        },
+        force: 'Experiment',
+      },
+      /* End internal overrides */
+      // {
+      //   variations: ['Control', 'Experiment'],
+      //   weights: [0.1, 0.9],
```
Comment (kmjennison) [inline graphql/database/experiments/features.js:462]: @spicermatthews Heads up, the weights should probably be 50/50.

File: graphql/database/users/__tests__/getLandingPagePhrase.test.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+describe('getLandingPagePhrase tests', () => {
+  it('returns constant phrase', async () => {
+    expect.assertions(1)
+    const getLandingPagePhrase = require('../getLandingPagePhrase').default
+
+    getUserFeature.mockResolvedValueOnce(
+      new Feature({
+        featureName: V4_SUPPORTING_STATEMENTS,
+        variation: 'Constant',
```
Comment (kmjennison) [inline graphql/database/users/__tests__/getLandingPagePhrase.test.js:22]: Nitpick: this variation should be "Control". No effect on the test behavior, though.

**Review by kmjennison — APPROVED** [review]
Great work here. LGTM! ☕ Added a few "FYI" comments.

File: graphql/database/users/__tests__/getLandingPagePhrase.test.js (hunk 2 above)
Comment (spicermatthews) [reply graphql/database/users/__tests__/getLandingPagePhrase.test.js]: Good catch. Fixed.

File: graphql/database/experiments/features.js (hunk 1 above)
Comment (spicermatthews) [reply graphql/database/experiments/features.js]: Good catch. Fixed.

**Pull request merged**